src/events.ts

```diff
@@ -192,24 +198,20 @@ const showQrCode = async (
   console.log(dataString)
 
   const base58encoded = await serializer.serialize(data)
-  console.log('base58encoded pair code', base58encoded)
+  console.log(base58encoded)
```

Comment:
@AndreasGassmann and this ones?

Reply:
Yes, we need this until we have a "copy to clipboard" feature. I added a comment.